doc/sfp-cmis/Interface-Link-bring-up-sequence.md

```diff
@@ -184,7 +185,235 @@ if transceiver is not present:
+
+    XCVRDMT -x APPL_DB : XCVRD main thread proecesses DEL event of APPL_DB PORT_TABLE
+    Note over XCVRDMT: generate_sigabrt = True
+    alt If threads > 0 are dead
+        XCVRDMT -x XCVRDMT : Kill XCVRD with SIGKILL
+    end
+    XCVRDMT -x CmisManagerTask : Stop CmisManagerTask
+    deactivate CmisManagerTask
+    XCVRDMT -x DomInfoUpdateTask : Stop DomInfoUpdateTask
+    deactivate DomInfoUpdateTask
+    XCVRDMT -x SfpStateUpdateTask : Stop SfpStateUpdateTask
+    deactivate SfpStateUpdateTask
+    Note over XCVRDMT : deinit()
+    alt self.sfp_error_event.is_set()
+        XCVRDMT -x XCVRDMT : sys.exit(SFP_SYSTEM_ERROR)
+    else if generate_sigabrt is True
+        XCVRDMT -x XCVRDMT : Kill XCVRD with SIGABRT
+
+    else
+        XCVRDMT -x XCVRDMT : Graceful exit
+    end
+    deactivate XCVRDMT
+```
+
+## Test plan and expectation
+|       Event      | APPL_DB cleared | Xcvrd restarted | Media renotify | MEDIA_SETTINGS_SYNC_DONE value on   xcvrd boot-up for initialized transceiver | CMIS re-init triggered | Link flap |
+|:----------------:|:---------------:|:---------------:|:--------------:|:-----------------------------------------------------------------------------:|:----------------------:|:---------:|
+| Xcvrd restart    | N               | Y               | N              | MEDIA_SETTINGS_DONE                                                           | N                      | N         |
+| Pmon restart     | N               | Y               | N              | MEDIA_SETTINGS_DONE                                                           | N                      | N         |
+| Swss restart     | Y               | Y               | Y              | MEDIA_SETTINGS_DEFAULT                                                        | Y                      | Y         |
+| Syncd restart    | Y               | Y               | Y              | MEDIA_SETTINGS_DEFAULT                                                        | Y                      | Y         |
+| config   reload  | Y               | Y               | Y              | MEDIA_SETTINGS_DEFAULT                                                        | Y                      | Y         |
+| Cold reboot      | Y               | Y               | Y              | MEDIA_SETTINGS_DEFAULT                                                        | Y                      | Y         |
+| Config shut      | N               | N               | N              | MEDIA_SETTINGS_DONE                                                           | N                      | Y         |
+| Config no   shut | N               | N               | N              | MEDIA_SETTINGS_DONE                                                           | N                      | Y         |
+| Warm   reboot    | N               | Y               | N              | MEDIA_SETTINGS_DONE                                                           | N                      | N         |
 # Out of Scope 
```

Comment:
Please update "Out of scope" section as some of it's covered as part of updated HLD.